files/lodash.compat/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "bower",
+  "name": "lodash.compat",
+  "repo": "lodash/lodash",
+  "files": {
+    "include": ["**/*.js", "**/*.map"]
```

Comment:
I was just copying the file from the non-compat lodash folder. Since that version included both minified and non-minified, I made this one the same. I don't have a problem with the changes you suggested, did you want me to apply them?